From 870bd4f2468d9b0af9b71f1e192396b3e482924e Mon Sep 17 00:00:00 2001
From: Sebastian Bazley <sebb@apache.org>
Date: Tue, 6 Jan 2015 02:39:25 +0000
Subject: [PATCH] VALIDATOR-235 UrlValidator rejects url with Unicode
 characters in domain label or TLD Seems to be OK now, update changes

git-svn-id: https://svn.apache.org/repos/asf/commons/proper/validator/trunk@1649722 13f79535-47bb-0310-9956-ffa450edef68
---
 RELEASE-NOTES.txt       | 2 ++
 src/changes/changes.xml | 3 +++
 2 files changed, 5 insertions(+)

diff --git a/RELEASE-NOTES.txt b/RELEASE-NOTES.txt
index 95f485423..928a0de55 100644
--- a/RELEASE-NOTES.txt
+++ b/RELEASE-NOTES.txt
@@ -81,6 +81,8 @@ BUGS FROM PREVIOUS RELEASE
                      Thanks to Jukka Timonen
  * [VALIDATOR-339] - URLValidator fails validating domain names with a trailing period, which are valid.
                      Thanks to Alex Blume
+ * [VALIDATOR-235] - UrlValidator rejects url with Unicode characters in domain label or TLD
+                     Thanks to Brian Preuß
 
 IMPROVEMENTS OVER PREVIOUS RELEASE
 ===================================
diff --git a/src/changes/changes.xml b/src/changes/changes.xml
index a9ff34592..abbc4fe9e 100644
--- a/src/changes/changes.xml
+++ b/src/changes/changes.xml
@@ -43,6 +43,9 @@ The <action> type attribute can be add,update,fix,remove.
   <body>
 
   <release version="1.4.1" date="tba" description="1.4 Maintenance release">
+    <action issue="VALIDATOR-235" dev="sebb" type="fix" due-to="Brian Preuß">
+     UrlValidator rejects url with Unicode characters in domain label or TLD
+    </action>
     <action issue="VALIDATOR-339" dev="sebb" type="fix" due-to="Alex Blume">
      URLValidator fails validating domain names with a trailing period, which are valid.
     </action>
